src/clients/http_client/__tests__/http_client.test.ts

```diff
@@ -872,7 +898,7 @@ describe('HTTP client', () => {
         body: {errors: 'Error 403'},
         code: 403,
         statusText: 'Error 403',
-        headers: {'X-Text-Header': ['Error 403']},
+        headers: {'X-Text-Header': 'Error 403'},
```

Comment:
(3/3) I'm a bit concerned about removing the `[]` from the comparison to get this to pass ... but this is the format we're expecting/intending to be returned for headers?

Reply:
Now using `canonicalizeHeaders()` when queuing a mocked response.